esp/platform/espcontext.cpp

```diff
@@ -466,9 +466,6 @@ class CEspContext : public CInterface, implements IEspContext
     {
         if (m_txSummary)
         {
-            if (!m_hasException && (getEspLogLevel() <= LogNormal))
```

Comment:
we might want to consider re-instituting this conditional based on a lower esploglevel